### PR TITLE
rewrote mountpoint handling logic to use action.request.params.rootPath

### PR DESCRIPTION
**Pull request by stevengill**
Handles issue https://github.com/adobe/developer.adobe.com/issues/135

### CODE REVIEW

**Comment by filmaj** [comment]
Not sure if it is related to these changes but static assets hosted in docs content repos don't seem to be loading. To see this, try to load this URL locally: http://localhost:3000/launch/docs/getting-started-1/general-launch-configuration-and-settings/create-a-property.html

File: src/default_html.pre.js [hunk 1]
```diff
@@ -65,7 +65,8 @@ async function pre(payload, action) {
     DOMUtil.spectrumify(body);
```
Comment (filmaj) [inline src/default_html.pre.js]: i think we want to use `node` here too

**Comment by filmaj** [comment]
Actually now when I try to pull this in and run locally, I can't load either /xd/docs or /launch/docs. I see no content.

**Comment by stevengill** [comment]
Yeah I saw that static assets are not working. Will keep debugging that after this PR.

This PR is working for me fine. I can see launch/xd docs. As well as starter/docs.

**Comment by filmaj** [comment]
Looks like maybe it has something to do with my slow home internet. Ignore my comment re: mounted strains, though if you could update the code that spectrumifies the DOM, that'd be appreciated.

**Pull request merged**